cypress/e2e/cypress-wrapper-driver.ts

```diff
@@ -1,41 +1,19 @@
 import { CypressHelper } from "@shellygo/cypress-test-utils";
 
-export default class CypressWrapperDriver {
+export default class MaputnikCypressHelper {
```

Comment:
Can you change the file name as well please?

Reply:
sure